testutil/singlevm/cleanup.sh

```diff
@@ -3,12 +3,17 @@
 . ~/local/demo.sh
 
 ciao_gobin="$GOPATH"/bin
+ciao_host=$(hostname)
 sudo killall ciao-scheduler
 sudo killall ciao-controller
 sudo killall ciao-launcher
 sleep 2
 sudo "$ciao_gobin"/ciao-launcher --alsologtostderr -v 3 --hard-reset
-sudo iptables -D FORWARD -p all -i ciao_br -j ACCEPT
+sudo iptables -D FORWARD -i ciao_br -j ACCEPT
+sudo iptables -D FORWARD -i ciaovlan -j ACCEPT
+if [ "$ciao_host" == "singlevm" ]; then
+	sudo iptables -D FORWARD -i ens2 -j ACCEPT
```

Comment:
Right now I am making this assumption as we use a canned ubuntu 16 image. Hence the check for singlevm. But to make it more generic, it would be best to determine the exiting physical interface by checking the route to the internet.

Something like

`ip -o route get 8.8.8.8 | cut -d ' ' -f 5`

That would be more generic